src/Equinor.ProCoSys.IPO.ForeignApi/MainApi/CommPkg/MainApiCommPkgService.cs

```diff
@@ -62,5 +62,24 @@ public class MainApiCommPkgService : ICommPkgApiService
 
             return commPkgs;
         }
+
+        public async Task<IList<ProCoSysRfocOnCommPkg>> GetRfocGuidsByCommPkgNosAsync(
+            string plant,
+            string projectName,
+            IList<string> commPkgNos)
+        {
+            var url = $"{_baseAddress}CommPkg/RfocByCommPkgNos" +
+                      $"?plantId={plant}" +
+                      $"&projectName={WebUtility.UrlEncode(projectName)}" +
+                      $"&api-version={_apiVersion}";
+            foreach (var commPkgNo in commPkgNos)
+            {
+                url += $"&commPkgNos={commPkgNo}";
```

Comment:
This will fail if url exceed 2000 chars

Reply:
I know, I have handled this in the FillRfocGuidsCommandHandler, by chunking if neccessary. This was also how I did it when synchronizing last time. But I can change this to being handles in this method instead.